app/controllers/pages_controller.rb

```diff
@@ -3,12 +3,16 @@
 class PagesController < ApplicationController
   skip_before_action :authenticate, except: [:home]
 
-  def show
-    render template: "pages/#{params[:page]}"
+  def home
```

Comment:
I thought the decision was to get rid of this route? You can:
- delete this method from the controller
- delete `except: [:home]` from the `skip_before_action` in this controller
- delete `get "/home", to: "pages#home", as: :home` from `routes.rb`

You might have some references to `home_path` across the code, you'll then be able to change these to `root_path`.